tools/docker/web.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-set -e
+set -ex
 
 # Ensure latest NAV code is built
 mydir=$(dirname $0)
 "$mydir/build.sh"
 cd /source
 
 
-django-admin check && exec django-admin runserver --insecure 0.0.0.0:80
+django-admin check && exec django-admin runserver --insecure 0.0.0.0:8080
```

Comment:
Missing newline after?

Reply:
bah humbug #blames-editor

Reply:
Shouldn't pre-commit handle this for us though... *lesigh*

Reply:
probably... should be cleaned up in the latest push, though.

Reply:
Pre-commit doesn't handle it for us since we don't have the hook `end-of-file-fixer` added (like we have in Argus). Should I make a PR to add that?

Reply:
Do!